sdk/core/ci.yml

```diff
@@ -68,5 +68,3 @@ extends:
         safeName: azurecoresse
       - name: typespec-ts-http-runtime
         safeName: typespectshttpruntime
```

Comment:
@timovv heads up we'll be publishing docs for this package, not sure if that will create any work on you due to a rename